main.go

```diff
@@ -1,7 +1,6 @@
 package main
 
 import (
-	"code.google.com/p/gcfg"
```

Comment:
Aside from the `README` and the configuration file example, there are also references to the configuration file inside of the `Makefile`.